### PR TITLE
Add stylelint config to monorepo

### PR DESCRIPTION
**Pull request by staszewski**
Jira: [MEC-]

**Description**

Adds stylelint config to monorepo.

<details>
<summary><b>Screenshots</b></summary>

</details>

<hr />

**Self checks**

- [ ] Self-review done.
- [ ] Unit tests included.
- [ ] E2E tests included.
- [ ] Documentation included or updated.

### CODE REVIEW

File: packages/appnroll-stylelint-config/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
```
Comment (staszewski) [inline packages/appnroll-stylelint-config/index.js:1]: This config comes from mudita-center.

**Review by kamilxunai — APPROVED** [review]

File: packages/appnroll-stylelint-config/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "https://github.com/Appnroll/appnroll-libs/tree/master/packages/appnroll-stylelint-config"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ext .js"
+  },
+  "homepage": "https://github.com/Appnroll/appnroll-libs/tree/master/packages/appnroll-stylelint-config#readme",
+  "author": "appnroll",
+  "license": "MIT",
+  "devDependencies": {
```
Comment (mateuszkocz) [inline packages/appnroll-stylelint-config/package.json:17]: The package depends on them to work. Hence those deps are not "dev".

File: packages/appnroll-stylelint-config/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  processors: ["stylelint-processor-styled-components"],
+  extends: [
+    "stylelint-config-recommended",
+    "stylelint-config-styled-components",
+  ],
+  rules: {
+    "unit-whitelist": [
+      ["%", "em", "rem", "vw", "vh", "s", "ms", "deg", "vmin", "vmax", "fr"],
+      {
+        ignoreProperties: { px: ["media-feature-name"] },
```
Comment (kamilxunai) [inline packages/appnroll-stylelint-config/index.js:11]: why?

Comment (staszewski) [reply packages/appnroll-stylelint-config/index.js:11]: As I said this config comes from mudita center. If you have any thoughts/objections please comment here.

File: packages/appnroll-stylelint-config/package.json (hunk 2 above)
Comment (staszewski) [reply packages/appnroll-stylelint-config/package.json]: 86d38e34359ddcafc0fca21cc4d2a22373a4805a

**Review by mateuszkocz — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Review by e9aru — APPROVED** [review]

**Review by Graffiss — APPROVED** [review]

**Pull request merged**